test/RewardsStreamerMP.t.sol

```diff
@@ -2101,35 +2101,77 @@ contract RewardsStreamerMP_RewardsTest is RewardsStreamerMPTest {
 
     function testRewardsBalanceOf() public {
         assertEq(streamer.totalRewardsSupply(), 0);
-
+        uint256 year = 365 days;
         uint256 initialTime = vm.getBlockTimestamp();
 
         _stake(alice, 100e18, 0);
-        assertEq(streamer.rewardsBalanceOf(vaults[alice]), 0);
-        assertEq(streamer.rewardsBalanceOf(vaults[bob]), 0);
 
         vm.prank(admin);
-        streamer.setReward(1000e18, 10 days);
-        assertEq(streamer.rewardsBalanceOf(vaults[alice]), 0);
-        assertEq(streamer.rewardsBalanceOf(vaults[bob]), 0);
```

Comment:
Any reason we're removing these?

Reply:
it's just a refactor of the full test, in the next line I set the same reward but for 1 year, and we have more assertions